tests/telemetry/test_writer.py

```diff
@@ -204,16 +204,12 @@ def test_app_started_event_configuration_override(test_agent_session, run_python
     which is then sent by periodic()
     """
     code = """
-import logging
-logging.basicConfig()
-
+# most configurations are reported when ddtrace.auto is imported
 import ddtrace.auto
-
-# By default telemetry collection is enabled after 10 seconds, so we either need to
-# to sleep for 10 seconds or manually call _app_started() to generate the app started event.
-# This delay allows us to collect start up errors and dynamic configurations
-import ddtrace
-ddtrace.internal.telemetry.telemetry_writer._app_started()
+# report configurations not used by ddtrace.auto
+import ddtrace.settings.symbol_db
+import ddtrace.settings.dynamic_instrumentation
+import ddtrace.settings.exception_replay
```

Comment:
How was this passing before you added this fix? Doesn't this mean that the DI and exception replay configs were either missing if they weren't automatically generated?

Reply:
Telemetry configuraations are queued when `ddtrace.config` and envier configurations are initialized. This occurs when `ddtrace.settings.som´_module` is imported.  Accessing `dtrace.internal.telemetry` had the sideffect of importing di, exception replay, and, symbol_db config modules. Now that I removed line 216, I had to make these imports explicit.